assets/src/editor/events/ticket-assignments-manager/ticket-assignments-manager-modal.js

```diff
@@ -848,74 +860,46 @@ export default compose( [
 			'event_espresso'
 		),
 	} ),
+	withEditorDateEntities,
+	withEditorTicketEntities,
+	ifCondition( ( { editorOpen } ) => editorOpen ),
 	withSelect( ( select, ownProps ) => {
 		const {
-			editorOpen,
 			dateEntity,
-			allDateEntities,
+			dateEntities,
 			ticketEntity,
-			allTicketEntities,
+			ticketEntities,
```

Comment:
Notice that `dateEntities` and `ticketEntities` are now provided via the `withEditorDateEntities` and `withEditorTicketEntities` hocs in the composite.  Thus the `all*` variations don't need to be passed down from components the ticket assignments manager is attached to and it can be self-contained (only receiving things from parent components that affect it's output).

Reply:
> this file is still a mess (not your fault) as there's lots of logic that should be extracted into separate functional components and files, but these changes clean up a few things nicely

Agreed, but at least now we're better setup for future refactors that can touch a smaller part of the codebase 👍